recipe/build_libsigcpp.sh

```diff
@@ -1,20 +1,8 @@
 #!/usr/bin/env bash
 set -ex
 
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./build
+meson ${MESON_ARGS} --prefix="${PREFIX}" -Dlibdir=lib builddir .
 
-if [[ "$target_platform" == osx* ]]; then
-  # C++17 support
-  export CXXFLAGS="${CXXFLAGS} -std=c++1z"
-fi
-
-# configure, make, install, check
-./configure --prefix="${PREFIX}" \
-  || { cat config.log; exit 1; }
-
-make -j${CPU_COUNT}
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
-  make check
-fi
-make install
+ninja -C builddir -j${CPU_COUNT}
+ninja -C builddir test
```

Comment:
What happens here when cross compiling and there is no emulator?

Reply:
It would fail I guess. I will remove the line.

Reply:
What I wanted to say was that I was surprised that testing passed on cross compiling platform osx-arm64. Turns out that the tests are skipped if we are cross compiling. So, it's okay to have this here. Can you bring this line back?

Reply:
Done